src/test/scala/com/cognite/sdk/scala/v1/DataPointsTest.scala

```diff
@@ -103,9 +103,9 @@ class DataPointsTest extends SdkTest with DataPointsResourceBehaviors {
     val sums2 = aggregates2("sum")
     val stepInterpolation2 = aggregates2("stepInterpolation")
     sums2.head.value should equal(65415.958570785)
-    sums2.last.value should equal(253965.25002673318)
+    sums2.last.value should equal(252857.16150906676)
```

Comment:
Why did those change?

Reply:
I am not sure, but that it the last value that making the api call in postman returns.